docs/root/start/sandboxes/templating.rst

```diff
@@ -0,0 +1,112 @@
+.. _install_sandboxes_templating:
+
+Templating
```

Comment:
Hum, the title can be better? I'm not sure I get it at first.

Reply:
@dio Sure, I can rename the example.  Any suggestions?

Reply:
@dio the original reason for calling it `templating` was because I was thinking along the lines that it most resembled mustache templates and go templates.

Reply:
Hum, I'm thinking that the name of an example should be inlined to a use-case (e.g. front-proxy, service-to-service proxy) or envoy's feature. Here I think you're exploring [`direct_response`](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#envoy-api-msg-route-directresponseaction), which is almost a way to serve a static file? That way it will be easier for a user making sense of this sandbox. WDYT?

Reply:
ok, I think of a title that is more describing of the use-case.